src/services/ldap/strategies/deltaSyncUtils.js

```diff
@@ -34,19 +11,12 @@ const getModifiedFilter = (timestamp, attributeName = 'modifyTimestamp') =>
  * @example
  *    filterForModifiedEntities(); // => ''
  *    filterForModifiedEntities('20201020000000Z', '(objectClass=person)');
- *    // => '(&(objectClass=person)(|(!(modifyTimestamp=*))(modifyTimestamp>=20201020000000Z)))'
+ *    // => '(objectClass=person)'
  */
 const filterForModifiedEntities = (school, existingFilter = '') => {
```

Comment:
we should delete this method, if it is not doing anything anymore and replace its calls everywhere